app/test_main.py

```diff
@@ -1,3 +1,51 @@
+import pytest
+from typing import Any
 from app.main import get_human_age
 
-# write your code here
+
+class TestGetHumanAge:
+
+    @pytest.mark.parametrize(
+        "cat_age,dog_age,result",
+        [
+            (0, 0, [0, 0]),
+            (14, 14, [0, 0]),
+            (23, 23, [1, 1]),
```

Comment:
Add tests when you use different values for `cat_age` and `dog_age` to test that the output is not changed in case different values are received. Besides that, you could cover more ranges with this approach.